job-service-db/src/main/resources/v3.1.0/changelog.xml

```diff
@@ -48,10 +48,19 @@
                          path="procedures/getJobs.sql" />
     </changeSet>
 
-    <changeSet id="add_job_sorting" author="Gamma Team">
+    <changeSet id="add_job_sorting" author="Gamma Team" runOnChange="true">
         <createProcedure schemaName="public"
                          procedureName="get_jobs"
                          path="procedures/getJobs.sql" />
+    </changeSet>
+
+    <changeSet id="add_job_sorting_index" author="">
```

Comment:
Had to separate out this index creation to it's own `changeSet` so the `add_job_sorting` can be run on change to avoid checksum issues when the stored proc is modified in the future.

The `preCondition` is to guard against attempting to create the index on a 3.1.0 db where the old version of the script has already been applied.

Reply:
Set the author.